pom.xml

```diff
@@ -14,15 +14,47 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.1</version>
-                <dependencies>
-                    <dependency>
-                        <groupId>org.junit.platform</groupId>
-                        <artifactId>junit-platform-surefire-provider</artifactId>
-                        <version>1.2.0</version>
-                    </dependency>
-                </dependencies>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>7</source>
+                    <target>7</target>
+                </configuration>
             </plugin>
```

Comment:
To raczej nie jest do niczego potrzebne, nie ma takiej wersji jak java 7
https://app.javastart.pl/kurs/testowanie-jednostkowe/tj-konfiguracja-uruchamianie-testow/lekcja/tj-konfiguracja-intellij-maven